src/components/FplButton/FplButton.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+import { makeStyles, withStyles } from '@mui/styles';
+import Icon from '@mui/material/Icon';
+
+const SpecialButton = withStyles({
+    root: {
+        background: "linear-gradient(180deg, #8B2FF3 0%, #4E73F8 100%)",
```

Comment:
all colors to come from colorsConst.js in common place

Reply:
will cover in release/W2